_posts/2020-05-06-community-update.md

```diff
@@ -0,0 +1,205 @@
+
+<hr>
+
+## Flink Minor Releases
+
+### Flink 1.9.3
+
+The community released Flink 1.9.3, covering some outstanding bugs from Flink 1.9! You can find more in the [announcement blogpost]((https://flink.apache.org/news/2020/04/24/release-1.9.3.html)).
+
+### Flink 1.10.1
+
+Also in the pipeline is the release of Flink 1.10.1, already in the [RC voting](http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/VOTE-Release-1-10-1-release-candidate-2-td41019.html) phase. So, you can expect Flink 1.10.1 to be released soon!
+
+<hr>
+
+## New Committers and PMC Members
+
+The Apache Flink community has welcomed **3 PMC Members** and **2 new Committers** since the last update. Congratulations!
+
+### New PMC Members
+
+<div class="row">
+  <div class="col-lg-3">
+    <div class="text-center">
+      <img class="img-circle" src="https://avatars2.githubusercontent.com/u/6242259?s=400&u=6e39f4fdbabc8ce4ccde9125166f791957d3ae80&v=4" width="90" height="90">
+      <p><a href="https://twitter.com/dwysakowicz">Dawid Wysakowicz</a></p>
+    </div>
+  </div>
+  <div class="col-lg-3">
+    <div class="text-center">
+      <img class="img-circle" src="https://avatars1.githubusercontent.com/u/4971479?s=400&u=49d4f217e26186606ab13a17a23a038b62b86682&v=4" width="90" height="90">
+      <p><a href="https://twitter.com/HequnC">Hequn Chen</a></p>
+    </div>
+  </div>
+  <div class="col-lg-3">
+    <div class="text-center">
+      <img class="img-circle" src="https://avatars3.githubusercontent.com/u/12387855?s=400&u=37edbfccb6908541f359433f420f9f1bc25bc714&v=4" width="90" height="90">
+      <p>Zhijiang Wang</p>
+    </div>
+  </div>
+</div>
+
+### New Committers
+
+<div class="row">
+  <div class="col-lg-3">
+    <div class="text-center">
+      <img class="img-circle" src="https://avatars3.githubusercontent.com/u/11538663?s=400&u=f4643f1981e2a8f8a1962c34511b0d32a31d9502&v=4" width="90" height="90">
+      <p><a href="https://twitter.com/snntrable">Konstantin Knauf</a></p>
+    </div>
+  </div>
+  <div class="col-lg-3">
+    <div class="text-center">
+      <img class="img-circle" src="https://avatars1.githubusercontent.com/u/1891970?s=400&u=b7718355ceb1f4a8d1e554c3ae7221e2f32cc8e0&v=4" width="90" height="90">
+      <p><a href="https://twitter.com/sjwiesman">Seth Wiesman</a></p>
+    </div>
+  </div>
+</div>
+
+<hr>
+	
+# The Bigger Picture
+
+## A new self-paced Apache Flink training
+
+<div style="line-height:60%;">
+    <br>
+</div>
+
+This week, the Flink website received the contribution of a self-paced training course curated by David ([@alpinegizmo](https://twitter.com/alpinegizmo)). The new materials guide you through the very basics of Flink and the DataStream API, and round off every concepts section with hands-on exercises to help you better assimilate what you learned.
```

Comment:
```suggestion
This week, the Flink website received the contribution of a self-paced training course curated by David ([@alpinegizmo](https://twitter.com/alpinegizmo)). The new materials guide you through the very basics of Flink and the DataStream API, and round off every concept section with hands-on exercises to help you better assimilate what you learned.
```

Reply:
I used an attributive noun on purpose here and personally prefer that.